src/python/antares_xpansion/input_checker.py

```diff
@@ -442,14 +578,9 @@ def check_options(options):
 
     option_items = options.items()
     for (option, value) in option_items:
-        if not check_setting_option_type(option, value):
-            flushed_print(
-                "check_settings : value %s for option %s has the wrong type!" % (value, option))
-            sys.exit(1)
         check_setting_option_value(option, value)
 
-    if options.get('yearly-weights', "") != "":
-        if options.get("cut_type") == "average":
-            flushed_print(
-                "check_settings : yearly-weights option can not be used when cut_type is average")
-            sys.exit(1)
+    if options.get('yearly-weights', "") != "" and options.get("cut_type") == "average":
```

Comment:
Remove this as `cut_type` is no longer used

Reply:
removed